handlers/search.go

```diff
@@ -267,7 +265,42 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 				WHERE from_address_text LIKE $1 || '%'
 			) a 
 			GROUP BY from_address_text`, lowerStrippedSearch)
-
+	case "count_indexed_validators_by_withdrawal_credential":
+		var ensData *types.EnsDomainResponse
+		if utils.IsValidEnsDomain(search) {
+			ensData, _ = GetEnsDomain(search)
+			if len(ensData.Address) > 0 {
+				lowerStrippedSearch = strings.ToLower(strings.Replace(ensData.Address, "0x", "", -1))
+			}
+		}
```

Comment:
**issue:** I don't know of a connection between withdrawal credentials and ENS names so this block looks like it should be deleted.
It makes sense in the search above `count_indexed_validators_by_eth1_address` because when you search for an ENS name you first want to convert it to the address it represents and then search for that address.

Reply:
In my implementation, the user can type a credential or _an address_ that validators withdraw to. A 0x01 credential is just a withdrawal address with a lot of 0s before, so I found it relevant to offer both possibilities.
So my intention here is to allow users to type an ENS domain to find the validators which withdraw to the underlying address.

If you confirm that I am wrong, I will remove this.

Reply:
I think the short answer here is just that we don't want that (for ENS and addresses).
Like the comment in the BIDS says we just want to show new search results if the user inputs exactly the 64 characters for withdrawal credentials.

If it is an ENS or address we already show results using the case above it `count_indexed_validators_by_eth1_address`.